src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -48,6 +49,29 @@ public abstract class TagHelperContent : IEnumerable<string>
         public abstract TagHelperContent Append(string value);
 
         /// <summary>
+        /// Appends formatted string by replacing each format item in the specified string 
+        /// with the text equivalent of a corresponding object's value.
+        /// </summary>
+        /// <param name="format">
+        /// The composite format <see cref="string"/> (see http://msdn.microsoft.com/en-us/library/txafckwd.aspx).
+        /// </param>
+        /// <param name="args">The object array to format.</param>
+        /// <returns>A reference to this instance after the append operation has completed.</returns>
+        public abstract TagHelperContent AppendFormat(string format, params object[] args);
```

Comment:
this overload requires an array allocation every time it's called.  suggest including the same overloads as `string.Format()`.  currently missing:
``` c#
public abstract TagHelperContent AppendFormat(string format, object arg0);
public abstract TagHelperContent AppendFormat(string format, object arg0, object arg1);
public abstract TagHelperContent AppendFormat(string format, object arg0, object arg1, object arg2);
```